components/layout/flow/inline/construct.rs

```diff
@@ -85,6 +85,15 @@ pub(crate) struct InlineFormattingContextBuilder {
     /// [to be split]: https://www.w3.org/TR/CSS2/visuren.html#anonymous-block-level
     block_in_inline_splits: Vec<Vec<ArcRefCell<InlineItem>>>,
 
+    /// If the [`InlineBox`] of an inline-level element is not damaged, it can be reused
+    /// to support incremental layout. Since an [`InlineBox`] can be split by block
```

Comment:
"Since" introduces a subordinated clause, but here there wouldn't be an independent clause owning it.

```suggestion
    /// to support incremental layout. An [`InlineBox`] can be split by block
```